site/content/en/docs/drivers/podman.md

```diff
@@ -7,23 +7,17 @@ aliases:
 
 ## Overview
 
-{{% pageinfo %}}
-This driver is experimental and in active development. Help wanted!
-{{% /pageinfo %}}
-
 The podman driver is an alternative container runtime to the [Docker]({{< ref "/docs/drivers/docker.md" >}}) driver.
 
 ## Requirements
 
-- Linux or macOS operating systems on amd64 architecture
```

Comment:
This "or macOS" was left since before, but the (wrong) code was removed in d0260cca998d59b293a287027230f845c5957a9c